src/layout/layout.jsx

```diff
@@ -0,0 +1,17 @@
+import { Footer } from "../footer";
+import { NavBar } from "../navbar";
+import { useGetUser } from "../data/useGetUser";
+
+export const Layout = ({ children }) => {
+  const { data } = useGetUser();
+  const { email, profileImage } = data || {};
+  const profile = data ? { email, profileImage } : null;
+
```

Comment:
이 부분은 다음과 같이 수정할 수 있을것 같습니다.
```suggestion
  const { data: profile } = useGetUser();
```